themes/10up-theme/includes/blocks.php

```diff
@@ -42,19 +42,46 @@ function setup() {
 }
 
 /**
- * Add in blocks that are registered in this theme
+ * Automatically registers all blocks that are located within the includes/blocks directory
  *
  * @return void
  */
 function register_theme_blocks() {
 	// Filter the plugins URL to allow us to have blocks in themes with linked assets. i.e editorScripts
 	add_filter( 'plugins_url', __NAMESPACE__ . '\filter_plugins_url', 10, 2 );
 
-	// Require custom blocks.
-	require_once TENUP_THEME_BLOCK_DIR . '/example-block/register.php';
+	// Register all the blocks in the theme
+	$block_directory_path = TENUP_THEME_BLOCK_DIR;
```

Comment:
Not a huge deal, but you could just be using the constant here. No need to declare it to a variable.

Reply:
Resolved.